sklearn/preprocessing/data.py

```diff
@@ -480,6 +480,12 @@ def minmax_scale(X, feature_range=(0, 1), axis=0, copy=True):
 class StandardScaler(BaseEstimator, TransformerMixin):
     """Standardize features by removing the mean and scaling to unit variance
 
+    The standard score of a data point `x` is calculated as:
+
+        z = (x - mean_) / scale_
+
+    But the formula can be different if `with_mean=False` or `with_std=False`.
```

Comment:
I see two problems here. (1) There is actually one mean_ and one scale_ per column. The documentation should make this explicit. 
(2) The documentation should say what is the different formula that is used when `with_mean = False` and `with_std = False`, and when both are false.

Reply:
I changed `mean_` to `u` and `scale_` to `std` to indicate this is more a high level explanation and not an actual implementation. I have also added their respective values when `with_mean=False` or `with_std=False`.

I agree that we have one `mean_` and one `scale_` per column, but this is covered at least partially in the next paragraph: `Centering and scaling happen independently on each feature by computing the relevant statistics on the samples in the training set.`